app/views/user/check_email.html.slim

```diff
@@ -0,0 +1,20 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds-from-desktop
+    h1.govuk-heading-l Check your email
+
+    p.govuk-body
+      | We have sent you an email with instructions for how to verify your email address.  We sent the email to:
+
+    <p class="govuk-body govuk-!-font-weight-bold">
+      = @user.email
+    </p>
```

Comment:
It seem weird to mix in some HTML like this. Should this be something like:
```
p.govuk-body
  | We have sent you an email with instructions for how to verify your email address.  We sent the email to:
    p.govuk-body.govuk-!-font-weight-bold = @user.email
```

Reply:
that didn't work when I tried it that way (slim didn't recognise the `!` as a proper character for a class)

Reply:
What about `class="thing-with-!"`